### PR TITLE
chore: update api document for 0.11.0

### PR DESCRIPTION
**Pull request by Jerome1210**

### CODE REVIEW

File: StraaSPlayerSDK/Classes/STSSDKPlayerView.html [hunk 1]
```diff
@@ -1257,6 +1257,89 @@
...
+                    </code>
+                  </div>
+                  <div class="height-container">
+                    <div class="pointer-container"></div>
+                    <section class="section">
+                      <div class="pointer"></div>
+                      <div class="abstract">
+                        <p>Gets the current time of current media.</p>
+
+<p>@discussion currentTime means the media playing time. At the beginning of a media, this value would be 0.
```
Comment (lollyrain) [inline StraaSPlayerSDK/Classes/STSSDKPlayerView.html]: Is "@discussion"  superfluous?(The description of `currentLiveDateTime` has the same issue.)

File: StraaSMessagingSDK/Classes/STSChatManager.html [hunk 2]
```diff
@@ -1438,6 +1441,242 @@
...
+                            <tr>
+                              <td>
+                                <code>
+                                <em>key</em>
+                                </code>
+                              </td>
+                              <td>
+                                <div>
+                                  <pre class="highlight plaintext"><code>   The key of metadata. The key name must match the regular expression format `^[A-Za-z0-9][A-Za-z0-9_
+</code></pre>
```
Comment (harryhsu) [inline StraaSMessagingSDK/Classes/STSChatManager.html]: The formatting of line 1511~1512 seems to be off.

File: StraaSPlayerSDK/Classes/STSSDKPlayerView.html (hunk 1 above)
Comment (Jerome1210) [inline StraaSPlayerSDK/Classes/STSSDKPlayerView.html]: fix in 8459298

File: StraaSMessagingSDK/Classes/STSChatManager.html (hunk 2 above)
Comment (Jerome1210) [inline StraaSMessagingSDK/Classes/STSChatManager.html]: fix in d78d8a5

**Pull request merged**